files/ru/web/api/web_animations_api/using_the_web_animations_api/index.html

```diff
@@ -319,7 +319,7 @@ <h2 id="Обратные_вызовы_и_обещания">Обратные вы
 <p>CSS Animations и Transitions (переходы) имеют свои события и они также могут быть воспроизведены и в Web Animations API:</p>
 
 <ul>
- <li><code><a href="/en-US/docs/Web/API/Animation/onfinish">onfinish</a></code> это обработчик событий для завершения события и он может быть запущена вручную в функции/команды <code><a href="/en-US/docs/Web/API/Animation/finish">finish()</a></code>.</li>
+ <li><code><a href="/en-US/docs/Web/API/Animation/onfinish">onfinish</a></code> это обработчик событий для завершения события и он может быть запущен вручную в функции/команды <code><a href="/en-US/docs/Web/API/Animation/finish">finish()</a></code>.</li>
```

Comment:
Поправил сразу перевод
```suggestion
 <li><code><a href="/en-US/docs/Web/API/Animation/onfinish">onfinish</a></code> это обработчик события завершения анимации, он может быть запущен вручную с помощью метода <code><a href="/en-US/docs/Web/API/Animation/finish">finish()</a></code>.</li>
```